sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -2,109 +2,105 @@
-    "Microsoft.Media.LiveEventIngestHeartbeat": models.MediaLiveEventIngestHeartbeatEventData,
-    "Microsoft.Media.LiveEventTrackDiscontinuityDetected": models.MediaLiveEventTrackDiscontinuityDetectedEventData,
-    "Microsoft.Resources.ResourceWriteSuccess": models.ResourceWriteSuccessData,
-    "Microsoft.Resources.ResourceWriteFailure": models.ResourceWriteFailureData,
-    "Microsoft.Resources.ResourceWriteCancel": models.ResourceWriteCancelData,
-    "Microsoft.Resources.ResourceDeleteSuccess": models.ResourceDeleteSuccessData,
-    "Microsoft.Resources.ResourceDeleteFailure": models.ResourceDeleteFailureData,
-    "Microsoft.Resources.ResourceDeleteCancel": models.ResourceDeleteCancelData,
-    "Microsoft.Resources.ResourceActionSuccess": models.ResourceActionSuccessData,
-    "Microsoft.Resources.ResourceActionFailure": models.ResourceActionFailureData,
-    "Microsoft.Resources.ResourceActionCancel": models.ResourceActionCancelData,
-    "Microsoft.ServiceBus.ActiveMessagesAvailableWithNoListeners":
-    models.ServiceBusActiveMessagesAvailableWithNoListenersEventData,
-    "Microsoft.ServiceBus.DeadletterMessagesAvailableWithNoListener":
-    models.ServiceBusDeadletterMessagesAvailableWithNoListenersEventData,
-    "Microsoft.Storage.BlobCreated": models.StorageBlobCreatedEventData,
-    "Microsoft.Storage.BlobDeleted": models.StorageBlobDeletedEventData,
-    "Microsoft.Storage.BlobRenamed": models.StorageBlobRenamedEventData,
-    "Microsoft.Storage.DirectoryCreated": models.StorageDirectoryCreatedEventData,
-    "Microsoft.Storage.DirectoryDeleted": models.StorageDirectoryDeletedEventData,
-    "Microsoft.Storage.DirectoryRenamed": models.StorageDirectoryRenamedEventData,
-    "Microsoft.Storage.LifecyclePolicyCompleted": models.StorageLifecyclePolicyCompletedEventData,
-    "Microsoft.Web.AppUpdated": models.WebAppUpdatedEventData,
-    "Microsoft.Web.BackupOperationStarted": models.WebBackupOperationStartedEventData,
-    "Microsoft.Web.BackupOperationCompleted": models.WebBackupOperationCompletedEventData,
-    "Microsoft.Web.BackupOperationFailed": models.WebBackupOperationFailedEventData,
-    "Microsoft.Web.RestoreOperationStarted": models.WebRestoreOperationStartedEventData,
-    "Microsoft.Web.RestoreOperationCompleted": models.WebRestoreOperationCompletedEventData,
-    "Microsoft.Web.RestoreOperationFailed": models.WebRestoreOperationFailedEventData,
-    "Microsoft.Web.SlotSwapStarted": models.WebSlotSwapStartedEventData,
-    "Microsoft.Web.SlotSwapCompleted": models.WebSlotSwapCompletedEventData,
-    "Microsoft.Web.SlotSwapFailed": models.WebSlotSwapFailedEventData,
-    "Microsoft.Web.SlotSwapWithPreviewStarted": models.WebSlotSwapWithPreviewStartedEventData,
-    "Microsoft.Web.SlotSwapWithPreviewCancelled": models.WebSlotSwapWithPreviewCancelledEventData,
-    "Microsoft.Web.AppServicePlanUpdated": models.WebAppServicePlanUpdatedEventData,
-}
+class SystemEventMappings(str, Enum):
```

Comment:
It might be worth adding some nodes about this class, like what's the map for, it maps from what to what -- some concepts.

because from a glance, I don't know how to use it :P